doc/tools/format-tool.md

```diff
@@ -27,9 +27,12 @@ the current project.
 
 `FormatTool` can be run in 3 modes:
 
-1. `FormatMode.overwrite`(default)
-2. `FormatMode.dryRun` (lists files that would be changed)
-3. `FormatMode.check` (dry-run _and_ sets the exit code if changes are needed)
+- `FormatMode.overwrite` (default)
+  - e.g. `dartfmt -w .`
+- `FormatMode.dryRun` (lists files that would be changed)
+  - e.g. `dartfmt -n .`
+- `FormatMode.check` (dry-run _and_ sets the exit code if changes are needed)
+  - e.g. `dartfmt -n --set-exit-if-changed .`
```

Comment:
It's funny, I was thinking about what the flags to `ddev format` would be for each of these modes, not the flags passed to the underlying tool.

Reply:
Oh, duh... that makes more sense :) I'll update